docs/fitting_types.rst

```diff
@@ -385,6 +385,28 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting moredimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like a temperature,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+At this example, we made a polynomial function with two coefficients,
+representing two terms of mixed order in x and y::
```

Comment:
x and y probably need some formatting. I'm not sure what the syntax is at the moment though.

Reply:
Do you mean something like a link or bold text?

Reply:
Italics or even better, mathtyped. I'm also not sure whether we did it in the rest of the docs.

Reply:
Italic is no problem, but I don't know the syntax of mathtyped text. Moreover I didn't notice any formatted text for variables in the docs. But this could be a new issue to work on ;)